spec/ameba/rule/lint/empty_ensure_spec.cr

```diff
@@ -61,7 +62,13 @@ module Ameba::Rule::Lint
       issue = s.issues.first
 
       issue.rule.should_not be_nil
-      issue.location.to_s.should eq "source.cr:2:3"
+      # TODO: refactor this in next release
+      # Crystal 1.4 changes the start location of Crystal::ExceptionHandler
+      if SemanticVersion.parse(Crystal::VERSION) <= SemanticVersion.parse("1.3.2")
+        issue.location.to_s.should eq "source.cr:2:3"
+      else
+        issue.location.to_s.should eq "source.cr:1:1"
```

Comment:
That's still incorrect. Code in question should take into account changes done in https://github.com/crystal-lang/crystal/pull/11476

Reply:
This is about to be refactored later (as the comment says). Out of scope of this PR.